docs/Makefile

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = python3.6 -msphinx
+SPHINXBUILD   = python -msphinx
```

Comment:
I needed to make this change because I'm on python 3.7

Reply:
Oh sure, thanks for changing this. :+1: